packages/styled-components/src/models/StyledComponent.js

```diff
@@ -173,7 +174,10 @@ function useStyledComponentImpl<Config: {}, Instance>(
     propsForElement.style = { ...attrs.style, ...props.style };
   }
 
-  propsForElement.className = Array.prototype
```

Comment:
completely unrelated nit, we know that `styledComponentId` is not an array and that `foldedComponentIds` is an array, so why aren't we using `foldedComponentIds.concat(styledComponentId)` or `[styledComponentId].concat(foldedComponentIds)` here?

Reply:
Ah yeah we could do that. I think the concat order used to be different before and a non-array was in the first position.